### DISCUSSION TITLE
one (flatline) rule for multiple queries

### DISCUSSION THREAD

**Discussion by mansior** [jertel/elastalert2#556] [Q&A] [answered]
Hey,

we have multiple small services that write their data into one elastic index:

2 example documents in elastic:
```
{
   "service_name": "first_service",
   "service_msg": "doing x"
},
{
   "service_name": "second_service",
   "service_msg": "doing something else"
}
```

I would like to be alerted as soon as any service stops to create documents in elastic for X hours.

Is it possible to achieve this without writing a separate rule(different query/filter) for each service?

Thanks

**Comment by jertel** [comment, marked as answer]
Yes, see https://elastalert2.readthedocs.io/en/latest/ruletypes.html?highlight=flatline#query-key.

**Reply by mansior** [reply]
Worked, thanks.
An important point would be NOT to use `use_count_query: true` ;)
